### DISCUSSION TITLE
Class setting on DataGridRow Command

### DISCUSSION THREAD

**Discussion by sanguinal** [Megabit/Blazorise#2064] [Q&A]
Hi All,

Not sure if this intentional or not, which is why i have not posted as a bug, but the save and cancel column when editing a row does not have the class set on this.

This appears to be due to the _DataGridRowEdit.Razor file when using the _DataGridRowCommand, it doesn't provide the class as a property when calling. If Class="@column.CellClass?.Invoke(Item)" is added this now works.

E.g.

        @foreach ( var column in Columns )
        {
            @if ( column.ColumnType == DataGridColumnType.Command )
            {
                @if ( ParentDataGrid.IsCommandVisible )
                {
                    <_DataGridRowCommand **Class="@column.CellClass?.Invoke(Item)"**  TItem="TItem" Item="@Item" EditState="DataGridEditState.Edit" Save="@SaveWithValidation" Cancel="@Cancel" Width="@column.Width" />
                }
            }
            else if ( column.ColumnType == DataGridColumnType.MultiSelect )
            {
                @if ( ParentDataGrid.MultiSelect )
                {
                    <TableRowCell></TableRowCell>

                }
            }
            else if ( column.Displayable )............and so on........

I hope this is helpful.

Kind Regards,

Sanguinal

**Comment by stsrki** [comment]
It might be a bug. Can you open a new ticket and I will investigate in detail once I have more time?

**Comment by sanguinal** [comment]
Done!

**Reply by stsrki** [reply]
Thanks
